<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpeechWidget</class>
 <widget class="QWidget" name="SpeechWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QCheckBox" name="autoPlaySpeechCheck">
     <property name="text">
      <string>自动播放语音</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="autoTransSpeechCheck">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;需要获取到token才可使用&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>自动语音转文字</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>百度AI Access Token</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>百度AI Secret Key</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>百度AI API Key</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="baiduSpeechApiKey"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="baiduSpeechSecretKey"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="baiduSpeechAccessTokenEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="refreshAccessTokenButton">
     <property name="text">
      <string>刷新 Access Token</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="styleSheet">
      <string notr="true">color: gray;</string>
     </property>
     <property name="text">
      <string>需要在&lt;a href=&quot;https://console.bce.baidu.com/ai&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;百度AI&lt;/span&gt;&lt;/a&gt;上注册账号并创建应用以获取 API Key 和 Secret Key
接着点击“刷新 Access Token”来获取 token。</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
